docs/example_manfest.yaml

```diff
@@ -0,0 +1,57 @@
+name: my-plugin
```

Comment:
thinking about renaming this to example-plugin

Reply:
renamed